lib/pg_online_schema_change/query.rb

```diff
@@ -301,6 +301,21 @@ def storage_parameters_for(client, table, reuse_trasaction = false)
         columns.first
       end
 
+      def view_definitions_for(client, table)
+        query = <<~SQL
+          select *
```

Comment:
I think we can probably do better than this query. We should be able to use `pg_depend` like this [postgres wiki article](https://wiki.postgresql.org/wiki/Pg_depend_display) details.

Reply:
ah nice! good call. Updated!